rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -7,8 +7,6 @@
   "required": [
     "$schema",
     "pid",
-    "barcode",
-    "call_number",
```

Comment:
But are these fields actually optional, I mean do we accept an item without barcode? If the barcode is generated by the system, then it is always filled in, it's not optional anymore. It only optional form the editor point of view.

It may be different with the call number, I don't know.

Reply:
According to me :
1. call_number 
- could be empty 
- shouldn't get system generated value.
- shouldn't be mandatory in the item editor.

2. barcode
- could be empty (because the librarian doesn't know it when it creates the item) 
- should get system generated value when it's empty
- shouldn't be mandatory in the item editor.

Reply:
@benerken It just seems weird to me that a field that has always a value is not required by the validation tool, ie the schema. I do understand that the librarian should be able to leave this field empty. I don't know if we can achieve that. If not, then the field we not be required, thus leaving the possibility to be saved without any barcode (at least theoretically). My question is, could we live with it?

Reply:
Item barcode is a required field populated by the current timestamp if not given.
Item call number is an optional field